integreat_cms/cms/forms/events/event_form.py

```diff
@@ -224,11 +224,6 @@ def clean(self) -> dict[str, Any]:
             self.data["start"] = cleaned_data["start"]
             self.data["end"] = cleaned_data["end"]
         logger.debug("EventForm validated [2] with cleaned data %r", cleaned_data)
```

Comment:
```suggestion
        has_not_location = cleaned_data['has_not_location']
        meeting_url = cleaned_data['meeting_url']
        id_location = cleaned_data['id_location']

        if has_not_location:
            cleaned_data['id_location'] = None
        else:
            cleaned_data['meeting_url'] = ""

        logger.debug("EventForm validated [2] with cleaned data %r", cleaned_data)
```
As mentioned below in `conditional-fields.ts` the value of `has_not_location` is indeed sent to the server, therefore I would suggest handling the setting of values for `meeting_url` and `id_location` here in the form.

Reply:
This is really weird. Apparently setting `cleaned_data["meeting_url"] = ""` is not enough, but manipulating `self.data` directly works for some reason.